src/validator.cpp

```diff
@@ -231,6 +225,54 @@ struct Validator::ValidatorImpl
      * @return @c true if @node is a supported MathML element and @c false otherwise.
      */
     bool isSupportedMathMLElement(const XmlNodePtr &node);
+
+    /**
+    * @brief Validate that equivalent variable pairs in the @p model
+    * have equivalent units.
+    * @cellml2_19 Validate that equivalent variable pairs in the @p model
+    * have equivalent units.
+    * Any errors will be logged in the @c Validator.
```

Comment:
What is that `@cellml2_19` about?

Reply:
Ah, well spotted! That's a leftover from where I was coding all the validator tests to make sure that everything in the spec was covered.